cts/scheduler/summary/unmanaged-promoted.summary

```diff
@@ -7,7 +7,6 @@ Current cluster status:
     * Clone Set: Fencing [FencingChild] (unmanaged):
       * FencingChild	(stonith:fence_xvm):	 Started pcmk-2 (unmanaged)
       * FencingChild	(stonith:fence_xvm):	 Started pcmk-1 (unmanaged)
-      * Stopped: [ pcmk-3 pcmk-4 ]
```

Comment:
All the other test output changes look good, but this file is questionable.

This one gets back to one of our original problem cases: pcmk-3/4 are offline, but their resource history shows FencingChild as started. I'm thinking it should be shown as started/unmanaged, and I'm not sure why it wasn't already, especially since non-clone resources like rsc_pcmk-3/4 are shown as started/unmanaged on them.

Reply:
It seems like the ping-1 and stateful-1 resources running on those nodes should also be shown as started/unmanaged and unpromoted/unmanaged respectively.  In pe__clone_text, they are both falling into the "else if (partially_active == FALSE)" block, which means print_full never gets set to TRUE, which means that block doesn't get called to print them out.